### PR TITLE
Update API docs to reflect the fact that federation is now also available…

### PR DESCRIPTION
**Pull request by RackerKate**
… for dedicated.

### CODE REVIEW

**Comment by RackerKate** [comment]
@stephamon Please review.

**Comment by stephamon** [comment]
@RackerKate will do!

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/build-46af944ba8/docs/rackspace-federation/)

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/build-46fad27867/docs/rackspace-federation/)

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/build-36dea3ab24/docs/rackspace-federation/). :bow:

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/build-2891ca87ed/docs/rackspace-federation/)

**Comment by RackerKate** [comment]
@cyrichardson Please review.

**Comment by cyrichardson** [comment]
@RackerKate, on it.

File: doc/gettingstarted/add-idp-cp.rst [hunk 1]
```diff
@@ -29,25 +30,24 @@ following table provides descriptions of these items:
    * - Item
      - Description
    * - Description
-     - Provide a description for your |idp|. This description is displayed in
+     - Provide a description for your |idp|. This description appears in
```
Comment (cyrichardson) [inline doc/gettingstarted/add-idp-cp.rst]: Provides a description

File: doc/gettingstarted/add-idp-cp.rst [hunk 2]
```diff
@@ -29,25 +30,24 @@ following table provides descriptions of these items:
...
        lists and in other areas in the Control Panel.
-   * - Login Domain
+   * - Email Domains
      - Provide a valid email domain, such as *mycompany.com*. Users provide
```
Comment (cyrichardson) [inline doc/gettingstarted/add-idp-cp.rst]: Provides a valid email domain

File: doc/attribmapping-basics/index.rst [hunk 3]
```diff
@@ -4,26 +4,26 @@
 Attribute mapping basics
```
Comment (cyrichardson) [inline doc/attribmapping-basics/index.rst]: Attribute Mapping basics

File: doc/attribmapping-basics/index.rst [hunk 4]
```diff
@@ -4,26 +4,26 @@
...
 ========================
 
-Attribute Mapping policies are YAML-formatted files that provide a means
+Attribute Mapping policies are YAML-formatted files that are used
```
Comment (cyrichardson) [inline doc/attribmapping-basics/index.rst]: Policies

File: doc/attribmapping-basics/index.rst [hunk 5]
```diff
@@ -4,26 +4,26 @@
...
 to map SAML attributes to Rackspace-required fields, such as roles and account
 permissions.
 
 An |amp| is required for every |idp| that you create.
 
 Attribute Mapping policies are composed of one or more rules. These rules
```
Comment (cyrichardson) [inline doc/attribmapping-basics/index.rst]: Policies

File: doc/attribmapping-basics/index.rst [hunk 6]
```diff
@@ -4,26 +4,26 @@
...
-Following are some common use cases for an |amp|:
+An |amp| has the following common use cases:
 
 - Assigning roles to a user based on a SAML attribute (such as *groups*
   provided by Active Directory Federation Services)
-- Identifying Rackspace-required values like *email address* when it is not
+- Identifying Rackspace-required values like *email address* when they are not
   stored in the standard SAML attribute location
-- Setting the expiration time that will require users to re-authenticate
+- Setting the expiration time after which users are required to re-authenticate
```
Comment (cyrichardson) [inline doc/attribmapping-basics/index.rst]: reauthenticate

File: doc/attribmapping-basics/index.rst [hunk 7]
```diff
@@ -4,26 +4,26 @@
...
-assign local values, which are attached to the user once they log in to
+assign local values that are attached to the user when they log in to
 Rackspace, based on explicit or remote values in the SAML exchange from your
 third-party provider.
 
-Following are some common use cases for an |amp|:
+An |amp| has the following common use cases:
 
 - Assigning roles to a user based on a SAML attribute (such as *groups*
   provided by Active Directory Federation Services)
```
Comment (cyrichardson) [inline doc/attribmapping-basics/index.rst]: Active Directory Federation Services (ADFS))

File: doc/attribmapping-basics/required-mapping.rst [hunk 8]
```diff
@@ -71,7 +70,7 @@ Setting values with Attribute Mapping
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 You can set values either explicitly or by using |amp| language features such
-as substitions or XPath.
+as substitutions or XPath.
 
 The following example syntax uses in-line substitutions in the local rule to
```
Comment (cyrichardson) [inline doc/attribmapping-basics/required-mapping.rst]: inline substitutions

File: doc/config-3p-saml/config-adfs/adfs-attribmapping.rst [hunk 9]
```diff
@@ -32,23 +33,27 @@ membership in your SAML attributes, see
...
+``rackspace-billing`` with users who you want to access Rackspace billing
 services by using the ``billing:admin`` Rackspace role.
 
-Notes:
+More information
+~~~~~~~~~~~~~~~~
 
-- You should change the ``groups`` specified in the example to match your
-  configured outgoing claim type for the active directory groups.
+When you mapping ADFS users to Rackspace roles or permissions, ensure that you
```
Comment (cyrichardson) [inline doc/config-3p-saml/config-adfs/adfs-attribmapping.rst]: you map ADFS

File: doc/config-3p-saml/config-adfs/adfs-attribmapping.rst [hunk 10]
```diff
@@ -32,23 +33,27 @@ membership in your SAML attributes, see
...
+perform the following tasks:
+
+- Change the ``groups`` specified in the example to match your
+  configured outgoing claim type for the ADFS Groups.
```
Comment (cyrichardson) [inline doc/config-3p-saml/config-adfs/adfs-attribmapping.rst]: ADFS groups.

File: doc/config-3p-saml/config-adfs/adfs-attribmapping.rst [hunk 11]
```diff
@@ -68,24 +73,24 @@ Notes:
              expire: PT4H
              # This would configure a maximum session duration of 4 hours, you may wish to set this to a SAML provided value
```
Comment (cyrichardson) [inline doc/config-3p-saml/config-adfs/adfs-attribmapping.rst]: four hours. You might want to set this to a SAML-provided

File: doc/config-3p-saml/config-adfs/adfs-attribmapping.rst [hunk 12]
```diff
@@ -68,24 +73,24 @@ Notes:
...
 
 
-Be sure to validate and modify the following items in your own |amp|:
+- Ensure that you validate and modify the following items in your own |amp|:
 
-- The **Active Directory** groups that users belong to and to which you want to
-  map specific Rackspace permissions.
-- The ``expire`` value/path
-- The ``email`` value/path
+  - The ADFS Groups that users belong to and to which you want to
```
Comment (cyrichardson) [inline doc/config-3p-saml/config-adfs/adfs-attribmapping.rst]: ADFS groups

File: doc/config-3p-saml/config-adfs/adfs-attribmapping.rst [hunk 13]
```diff
@@ -68,24 +73,24 @@ Notes:
...
+    map specific Rackspace permissions
+  - The ``expire`` value/path
```
Comment (cyrichardson) [inline doc/config-3p-saml/config-adfs/adfs-attribmapping.rst]: value or path

File: doc/config-3p-saml/config-adfs/adfs-attribmapping.rst [hunk 14]
```diff
@@ -68,24 +73,24 @@ Notes:
...
+  - The ``email`` value/path
```
Comment (cyrichardson) [inline doc/config-3p-saml/config-adfs/adfs-attribmapping.rst]: value or path

File: doc/attribmapping-basics/required-mapping.rst [hunk 15]
```diff
@@ -134,13 +133,12 @@ SAML providers, see :ref:`index-configuring-3p-saml-ug`.
            domain: "636462353"
            # Domain must be set to your Identity Domain
            name: "{D}"
-           #  Username will be set from element named "name" value in your SAML
+           #  Username is set from the element named "name" value in your SAML
            email: "{At(http://schemas.xmlsoap.org/ws/2005/05/identity/claims/emailaddress)}"
            #  Locates the attribute with the above URL as the claim type/name
```
Comment (cyrichardson) [inline doc/attribmapping-basics/required-mapping.rst]: type or name

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/build-2891ca87ed/docs/rackspace-federation/). :bow:

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/build-b51b5750c8/docs/rackspace-federation/). :bow:

**Comment by rackernexus** [comment]
:mag_right: [Preview](https://staging.developer.rackspace.com/build-34a7ae4460/docs/rackspace-federation/)

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/build-23a94ad724/docs/rackspace-federation/). :bow:

**Pull request merged**